Source/DafnyDriver/CompilerDriver.cs

```diff
@@ -596,6 +596,7 @@ private record TargetPaths(string Directory, string Filename) {
       if (moreText != null) {
         target.Write(moreText);
       }
+      target.Close();
```

Comment:
Does the above using directive not already guarantee that target is closed? This line seems obsolete.